File: app/src/main/java/xyz/zisarknar/keyboardlisu/keyboard/LisuKeyboard.java
package xyz.zisarknar.keyboardlisu.keyboard;

import android.content.Context;
import android.inputmethodservice.Keyboard;

public class LisuKeyboard extends Keyboard {

    public LisuKeyboard(Context context, int xmlLayoutResId) {
        super(context, xmlLayoutResId);
    }

    public LisuKeyboard(Context context, int xmlLayoutResId, int modeId, int width, int height) {
        super(context, xmlLayoutResId, modeId, width, height);
    }

    public LisuKeyboard(Context context, int xmlLayoutResId, int modeId) {
        super(context, xmlLayoutResId, modeId);
    }

    public LisuKeyboard(Context context, int layoutTemplateResId, CharSequence characters, int columns, int horizontalPadding) {
        super(context, layoutTemplateResId, characters, columns, horizontalPadding);
    }
}

File: app/src/main/java/xyz/zisarknar/keyboardlisu/keyboard/LisuKeyboard.kt
package xyz.zisarknar.keyboardlisu.keyboard

import android.content.Context
import android.inputmethodservice.Keyboard

class LisuKeyboard : Keyboard {

    constructor(context: Context, xmlLayoutResId: Int) : super(context, xmlLayoutResId) {}

    constructor(context: Context, xmlLayoutResId: Int, modeId: Int, width: Int, height: Int) : super(context, xmlLayoutResId, modeId, width, height) {}

    constructor(context: Context, xmlLayoutResId: Int, modeId: Int) : super(context, xmlLayoutResId, modeId) {}

    constructor(context: Context, layoutTemplateResId: Int, characters: CharSequence, columns: Int, horizontalPadding: Int) : super(context, layoutTemplateResId, characters, columns, horizontalPadding) {}
}
